KinoPoiskAutomatedTests/PageObjects/FilmPage.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using KinoPoiskAutomatedTests.WebDriver;
+using OpenQA.Selenium;
+
+namespace KinoPoiskAutomatedTests.PageObjects
+{
+    public class FilmPage : DefaultPage
+    {
+        private IWebElement _rating => driver.FindElement(By.XPath("//span[contains(@class,'rating')][1]"));
+        private IWebElement _traillerButton => driver.FindElement(By.XPath("//button[text()='Трейлер']/.."));
+        private By _trailerEnded => By.XPath("//div[contains(@class,'autoplay')][1]");
```

Comment:
Please, separate storing if different types of variables.

Reply:
resolved